### PR TITLE
Update info messages & docs - Fixes: #351 

### LINKED ISSUE #351: missing value in Default() info message
### 🐛 Describe the bug

Tiny bug in print, verbose=3. It shows" Default" instead of the value. :
```
LDM algorithm used. Defaulting broadening method from <radis.misc.utils.Default object at 0x0000018C126EA070> to Voigt
```

### 💡 Possible solutions

Could be corrected by setting __repr__ for the Default class

### PR DESCRIPTION
**Pull request by anandxkumar**
Fixes #351 and shifts `predict_time()` to `broaden_lines` and `broaden_lines_noneq`

### CODE REVIEW

File: radis/lbl/factory.py [hunk 1]
```diff
@@ -528,15 +528,18 @@ def __init__(
...
         if isinstance(broadening_method, Default):
-            if optimization in ("simple", "min-RMS") and broadening_method != "voigt":
+            if (
+                optimization in ("simple", "min-RMS")
+                and broadening_method.value != "voigt"
+            ):
                 if self.verbose >= 3:
                     printg(
-                        "LDM algorithm used. Defaulting broadening method from {0} to Voigt".format(
+                        "LDM algorithm used. Defaulting broadening method from {0} to 'voigt'".format(
```
Comment (erwanp) [inline radis/lbl/factory.py:537]: I think we can simplify the code : in both cases (DIT/not DIT) we use "voigt" anyway. Why should we even compare / print a message ? 

All of this block could be reduced to 
```
if isinstance(broadening_method, Default):
    broadening_method = broadening_method.value
```
and maybe we could even avoid using Default ?

File: radis/lbl/factory.py [hunk 2]
```diff
@@ -1716,7 +1719,7 @@ def _is_at_equilibrium():
                 raise NotImplementedError("broadening_method not implemented")
         elif optimization is None:
             if broadening_method == "voigt":
-                estimated_time = 6.6487e-08 * n_lines * truncation / wstep * factor
+                estimated_time = 6.6487e-08 * n_lines * truncation / wstep
```
Comment (erwanp) [inline radis/lbl/factory.py:1722]: is "factor" from L1677 still used anywhere else?

**Review by erwanp — CHANGES_REQUESTED** [review]

File: radis/lbl/factory.py (hunk 2 above)
Comment (anandxkumar) [reply radis/lbl/factory.py:1722]: No. It is being used by optimization =`simple` and `min-RMS`. For non-eq cases factor = 2 so we can simply refactor it as :
```
if not  _is_at_equilibrium:
   factor = 2
```
and remove rest of the code

File: radis/lbl/factory.py (hunk 1 above)
Comment (anandxkumar) [reply radis/lbl/factory.py:537]: Yes, Default was used for the purpose that we had different `broadening_methods` for DIT and LBL, but now we are using `voigt` as default so there is no need for `Default`.

**Review by erwanp — COMMENTED** [review]
Last thing : in def predict_time(self): docstrings, can you mentino what this function is doing ; and also reference some of your graphs (ideally the Tableau ones were you derived the predict function ), and your https://anandxkumar.github.io/Benchmark_Visualization_GSoC_2021  project ? 

Will be very helpful in the long run. 

You should also copy a similar text in the docs under [Performance](https://radis.readthedocs.io/en/latest/lbl/lbl.html#precompute-spectra) ; for instance create a section "Benchmark". We want people to be able to see your graphs !

**Comment by anandxkumar** [comment]
You can merge, the test has passed along linting! (Travis had connection issue as reported before)

**Review by erwanp — APPROVED** [review]
All good, I let you merge!

**Comment by erwanp** [comment]
Test failure is yet again a connection error with `test_eq_vs_noneq_isotope`.      see #350
You can ignore;

**Pull request merged**